RELEASE_NOTES.md

```diff
@@ -1,3 +1,8 @@
+### Version 1.1.0
+__Changes__
+- patched "bad status line" error with newer kb-sdk install of SDK_LOCAL clients
+- added generate_microbiome_in_silico_reads_from_real_reads() method
```

Comment:
I can't seem to find this function. Did you include this or did I just not find it?

Reply:
I got ahead of myself.  will add function next, but not as part of this PR